src/xrpld/overlay/detail/PeerImp.cpp

```diff
@@ -2757,6 +2767,15 @@ PeerImp::checkTransaction(
     // VFALCO TODO Rewrite to not use exceptions
     try
     {
+        // charge strongly for relaying batch txns
+        if (stx->isFlag(tfInnerBatchTxn))
```

Comment:
```suggestion
        if (stx->isFlag(tfInnerBatchTxn) && rules().enabled(featureBatch))
```

Reply:
There are no rules() in PeerImp so how would you like this to be handled?

Reply:
I believe there's a global `Rules` object now (trying to find it right now...)

Reply:
`getCurrentTransactionRules` is the function name

Reply:
Here's the commit, if that's helpful: https://github.com/XRPLF/rippled/commit/b65cea1984248ee5fa2fc02015914d543820902e